src/Input/InputLabel.js

```diff
@@ -18,6 +18,9 @@ export const styles = (theme: Object) => ({
     // slight alteration to spec spacing to match visual spec result
     transform: `translate(0, ${theme.spacing.unit * 3 - 1}px) scale(1)`,
   },
+  labelDense: {
+    transform: `translate(0, ${theme.spacing.unit * 2.5 - 1}px) scale(1)`,
```

Comment:
![capture d ecran 2017-08-29 a 15 39 38](https://user-images.githubusercontent.com/3165635/29823911-5ac73f8a-8cd0-11e7-9abe-2a4bc996e363.png)

For some reason, the following style better works:
```js
transform: `translate(0, ${theme.spacing.unit * 2.5 + 1}px) scale(1)`,
```

![capture d ecran 2017-08-29 a 15 41 02](https://user-images.githubusercontent.com/3165635/29823954-7d0f5618-8cd0-11e7-85bd-72cd888be2a4.png)


We could also add a comment around compensating for the `inputDense` style of the `Input` component